metadata_service/proxy/atlas_proxy.py

```diff
@@ -997,3 +997,34 @@ def get_resources_using_table(self, *,
                                   id: str,
                                   resource_type: ResourceType) -> Dict[str, List[DashboardSummary]]:
         return {}
+
+    def put_user(self, *, data: User) -> None:
```

Comment:
if the methods are solely for testing, do you think we could have a subclass Test class of different proxy in test file which extends it with the below methods?

Reply:
Oh, sure! That should work fine.

- [ ] some roundtrip testable subclass